src/languages/swift.js

```diff
@@ -96,7 +82,114 @@ export default function(hljs) {
         { begin: /\b0b([01]_*)+\b/ },
       ]
   };
-  SUBST.contains = [NUMBER];
+
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#grammar_string-literal
+  var ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\[0\\tnr"']|\\u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var RAW1_ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\#[0\\tnr"']|\\#u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var RAW2_ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\##[0\\tnr"']|\\##u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var RAW3_ESCAPED_CHARACTER = {
+    className: 'subst',
+    begin: /\\###[0\\tnr"']|\\###u\{[0-9a-fA-F]{1,8}\}/
+  };
+  var ESCAPED_NEWLINE = {
+    className: 'subst',
+    begin: /\\[\u0009\u0020]*(?:[\u000a\u000d]|\u000d\u000a)/
```

Comment:
Is this just `\r` and `\n`?  If so leets please use those common expressions instead.

I'm not sure what the 9 and 20 are? Can you clarify?